src/context/Baser.spec.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import assert from 'assert'
+import { mockVscode } from '../test/mockVscode' // Must come before Baser.
```

Comment:
>// Must come before Baser. [](start = 48, length = 26)

Why?

Reply:
`mockVscode` mocks/intercepts some other modules such as `vscode`.

I'd ask for some leeway for all test-related code. It's work-in-progress as well as strategy-in-progress. Experimenting with a few different approaches. Discussion is welcome, but ideally at the higher level.